core/templates/pages/topic-editor-page/modal-templates/create-new-story-modal.component.html

```diff
@@ -1,19 +1,23 @@
 <div class="modal-header create-new-story-header">
   <h3>
-    New Story
+    New story
   </h3>
+  <i class="fa fa-times modal-close-button" (click)="cancel()"></i>
 </div>
-<form ng-submit="confirm(story)">
+<form (submit)="save()">
```

Comment:
As you are inheriting from `ConfirmOrCancelModal`, you should use and override confirm function instead.

Reply:
Done